src/DataTable/DataTable.tsx

```diff
@@ -40,6 +40,10 @@ export interface DataTableProps<Columns, Rows> {
   className?: string
 }
 
+const stringifyIfNeeded = (value: any) => {
+  return typeof value === "boolean" ? String(value) : value
```

Comment:
```suggestion
  return (value === true || value === false) ? String(value) : value
```

would be faster, but this is optional thing

Reply:
Should we worry about other special values like, null, NaN, undefined, {}, []. Or TS won't let you use those?

Reply:
<img width="332" alt="Screenshot 2019-07-17 at 14 32 54" src="https://user-images.githubusercontent.com/179534/61375695-cdf28400-a89f-11e9-9436-311f289c0c3a.png">

Reply:
`null`, `undefined`, etc are valid React children, so I expect that to be handled in the userland. Original https://github.com/contiamo/operational-ui/pull/1132 was dealing with booleans only